packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation_objc/include/camera_avfoundation/FLTCam.h

```diff
@@ -42,10 +42,14 @@ NS_ASSUME_NONNULL_BEGIN
 @property(assign, nonatomic) BOOL isRecording;
 @property(assign, nonatomic) BOOL isRecordingPaused;
 @property(strong, nonatomic, nullable) NSObject<FLTAssetWriter> *videoWriter;
-@property(assign, nonatomic) BOOL videoIsDisconnected;
-@property(assign, nonatomic) BOOL audioIsDisconnected;
-@property(assign, nonatomic) CMTime videoTimeOffset;
-@property(assign, nonatomic) CMTime audioTimeOffset;
+/// Whether the recording is disconnected.
+@property(assign, nonatomic) BOOL isRecordingDisconnected;
+/// Represents sum of all recording pauses/interruptions.
+@property(assign, nonatomic) CMTime recordingTimeOffset;
+/// Output to use for adjusting of recording offset.
```

Comment:
can you add comment on what "adjusting of recording offset" means and why we need to adjust the offset?

Reply:
It should have been "adjusting of recording time offset" -> `recordingTimeOffset` which as mentioned is sum of all pauses in recording so adjusting it of course means that if there is another pause then add duration of that pause into that offset. So after that offset is applied then we get continuous recording.